src/olympia/reviewers/forms.py

```diff
@@ -370,19 +382,19 @@ def __init__(self, *args, **kw):
         self.helper = kw.pop('helper')
         super(ReviewForm, self).__init__(*args, **kw)
 
-        # Info request deadline needs to be readonly unless we're an admin.
+        # Delayed rejection period needs to be readonly unless we're an admin
         user = self.helper.handler.user
-        deadline_widget_attributes = {}
-        info_request_deadline = self.fields['info_request_deadline']
+        rejection_period_widget_attributes = {}
+        rejection_period = self.fields['delayed_rejection_days']
         if not acl.action_allowed_user(user, amo.permissions.REVIEWS_ADMIN):
-            info_request_deadline.min_value = info_request_deadline.initial
-            info_request_deadline.max_value = info_request_deadline.initial
-            deadline_widget_attributes['readonly'] = 'readonly'
-        deadline_widget_attributes.update({
-            'min': info_request_deadline.min_value,
-            'max': info_request_deadline.max_value,
-        })
-        info_request_deadline.widget.attrs.update(deadline_widget_attributes)
+            rejection_period.min_value = rejection_period.initial
+            rejection_period.max_value = rejection_period.initial
+            rejection_period_widget_attributes['readonly'] = 'readonly'
+        rejection_period_widget_attributes['min'] = rejection_period.min_value
+        rejection_period_widget_attributes['max'] = rejection_period.max_value
```

Comment:
If we disable the widget, then the value isn't sent via POST, plus it looks kinda weird in the UI (it will show the greyed out input which may make the value hard to read)